Automattic/jetpack
# Newsletters: Add Paywall block settings

Add Newsletter settings panel to Paywall Block settings

<img width="1213" alt="Screenshot 2023-08-04 at 11 49 34" src="https://github.com/Automattic/jetpack/assets/104869/8297fb25-290a-4fa0-a1d2-72b09059a119">


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* `define( 'JETPACK_BLOCKS_VARIATION', 'beta' );`
* Add a paywall block and make sure Newsletter visibility can be managed from block settings.
